target/linux/ramips/image/mt7620.mk

```diff
@@ -244,6 +244,15 @@ define Device/u25awf-h1
 endef
 TARGET_DEVICES += u25awf-h1
 
+define Device/u35wf
```

Comment:
Have you actually tested a sysupgrade? It doesn't work. Please use `define Device/kimax_u35wf` or `define Device/kimax_u35awf` depending on what you intended to do with the compatible string.